code/obj/item/clothing/plants.dm

```diff
@@ -26,34 +27,41 @@
 	desc = "Bird of Paradise flowers, or Crane Flowers, are named for their resemblance to the ACTUAL birds of the same name. Both look great sitting on your head either way."
 	icon_state = "flower_bop"
 	item_state = "flower_bop"
+	can_bouquet = TRUE
+
+	New()
+		. = ..()
+		src.AddComponent(/datum/component/bouquet, can_bouquet)
 
 /obj/item/clothing/head/flower/hydrangea
 	name = "hydrangea"
 	desc = "Hydrangeas are popular ornamental flowers due to their colourful, pastel flower arrangements; this one has been trimmed nicely for wear as an accessory."
 	icon_state = "flower_hyd"
 	item_state = "flower_hyd"
 
-/obj/item/clothing/head/flower/hydrangea/pink
-	name = "pink hydrangea"
-	icon_state = "flower_hyd-pink"
-	item_state = "flower_hyd-pink"
+	pink
+		name = "pink hydrangea"
+		icon_state = "flower_hyd-pink"
+		item_state = "flower_hyd-pink"
 
-/obj/item/clothing/head/flower/hydrangea/blue
-	name = "blue hydrangea"
-	icon_state = "flower_hyd-blue"
-	item_state = "flower_hyd-blue"
+	blue
+		name = "blue hydrangea"
+		icon_state = "flower_hyd-blue"
+		item_state = "flower_hyd-blue"
 
-/obj/item/clothing/head/flower/hydrangea/purple
-	name = "purple hydrangea"
-	icon_state = "flower_hyd-purple"
-	item_state = "flower_hyd-purple"
+	purple
+		name = "purple hydrangea"
+		icon_state = "flower_hyd-purple"
+		item_state = "flower_hyd-purple"
```

Comment:
Why?

Reply:
oh huh, forgot about that. That's left over from when i atomised out another pr, my bad. I'll ditch it